ui/user.go

```diff
@@ -0,0 +1,38 @@
+package ui
+
+import (
+	"net/http"
+
+	"github.com/OLIENTTECH/backend-challenges/pkg/log"
+	"github.com/OLIENTTECH/backend-challenges/usecase"
+	"github.com/labstack/echo/v4"
+)
+
+type User interface {
+	GetListUsers(c echo.Context) error
```

Comment:
[must] GetとListは意味同じ（１つとるか，複数とるか）なので，混合したら🙅‍♀️です！
下記参考になるかと👀
https://cloud.google.com/apis/design/standard_methods?hl=ja
```suggestion
type User interface {
	ListUsers(c echo.Context) error
```